### ISSUE TITLE
[windows-11-start-menu-styler] Start menu looks messed up, somehow?

### ISSUE THREAD

**Issue by naokoshoto** [ramensoftware/windhawk-mods#1297]
I don't know why but bunch of prebundled themes on this mods are visually broken

![image](https://github.com/user-attachments/assets/970af3af-855b-4643-82ee-3ed9509bd5b1)

Windows Version: Windows 11 version 24H2 build 26120.2415

Logs: https://justpaste.it/g8a8d

**Comment by m417z** [comment]
26120 is a preview build. Preview builds (Dev, Canary, Beta, Release Preview) aren't generally supported. They're a moving target, some things might work, and other things might be incompatible. For best compatibility with Windhawk, avoid using preview Windows builds.

In this case, most likely the theme has to be adjusted to the changes in the new Windows build.

**Comment by naokoshoto** [comment]
Any workaround regarding this? Maybe disabling some flags on vivetool or something?

**Comment by xellotath** [comment]
Also  having this, on latest dev channel. But for me anything that tries to display all apps just won't show them at all. I know it's not supported, but if you can think of where the culprit might be I can go into it and mess with it myself. Generally I just want start to go on all apps by default instead of pinned and recommended, don't care about style so much.

**Comment by m417z** [comment]
@xellotath you might want to try this mod as an alternative: [Show all apps by default in start menu](https://windhawk.net/mods/start-menu-all-apps).

**Comment by xellotath** [comment]
@m417z My bad I forgot to mention, I tried that one already. Breaks the start menu, it doesn't come up at all with it enabled.

**Comment by xellotath** [comment]
26200.5103

**Comment by m417z** [comment]
There are no online results for this build, is there a mistake? In any case, if it's an old build, updating might help. And if not, then unfortunately preview builds aren't a priority as I mentioned in my first comment here.

**Comment by xellotath** [comment]
No I just can't read numbers apparently 26200.5603, just the latest dev build. Sorry I'm very tired 😂

**Comment by m417z** [comment]
Is this issue different from https://github.com/ramensoftware/windhawk-mods/issues/1786? Can you try this workaround? https://github.com/ramensoftware/windhawk-mods/issues/1786#issuecomment-2920787738

**Comment by xellotath** [comment]
Yeah the all apps scroll down just sounds like it would break most styles. I'll get used to the desfult style in the meantime 🥲

**Comment by Andy76swe** [comment]
Any solution?

**Comment by m417z** [comment]
@Andy76swe for preview builds, there's no solution, and it's not a priority as described in my first comment here.

For non-preview builds, the mod option to revert to the old layout might help.

**Comment by m417z** [comment]
Closing as this issue is about old preview builds, and is probably no longer relevant.
